dbdreader/dbdreader.py

```diff
@@ -485,20 +485,20 @@ def __init__(self):
                        'the8x3_filename':'string'}
         self.info={}
 
-    def read_header(self,fp):
+    def read_header(self, fp, filename=''):
         ''' read the header of the file, given by fp '''
         fp.seek(0)
         if not self.parse(fp.readline())=='dbd_label':
-            raise ValueError("Seems not to be a valid DBD file")
+            raise ValueError(f"{filename} seems not to be a valid DBD file")
         n_read=1
-        while 1:
+        while True:
             self.parse(fp.readline())
             n_read+=1
             if 'num_ascii_tags' in self.info  and \
                self.info['num_ascii_tags']==n_read:
                 break
         if self.info['encoding_ver']!=ENCODING_VER:
-            raise ValueErro('Incompatible encoding version detected.')
```

Comment:
Note I think this was a bug on master...

Reply:
Bug went undetected because it seems the encoding version has never been changed.